---
id: data-access
title: 20. Data Access
checklist: 
    text: Provide a statement describing if and where study data are available.
---

## What to write

Provide a statement describing if and where study data are
available.

## Explanation

A data-sharing statement describes how others can
access the data on which the paper is based. Sharing adequately
annotated data allows others to replicate data analyses so that results
can be independently tested and verified. Data sharing allows the data
to be repurposed and new datasets to be created by combining data from
multiple studies (e.g., to be used in secondary analyses). This allows
others to explore new topics and increases the impact of the study,
potentially preventing unnecessary use of animals and providing more
value for money. Access to raw data also facilitates text and automated
data mining [@pbio.3000411.ref246].

An increasing number of publishers and funding bodies require authors or
grant holders to make their data publicly available
[@pbio.3000411.ref247]. Journal articles with accompanying data may
be cited more frequently
[@pbio.3000411.ref248; @pbio.3000411.ref249]. Datasets can also be
independently cited in their own right, which provides additional credit
for authors. This practice is gaining increasing recognition and
acceptance [@pbio.3000411.ref250].

When possible, make available all data that contribute to summary
estimates or claims presented in the paper. Data should follow the FAIR
guiding principles [@pbio.3000411.ref251]; that is, data are
findable, accessible (i.e., do not use outdated file types),
interoperable (can be used on multiple platforms and with multiple
software packages), and reusable (i.e., have adequate data descriptors).

Data can be made publicly available via a structured, specialised
(domain-specific), open-access repository such as those maintained by
the National Center for Biotechnology Information (NCBI,
<https://www.ncbi.nlm.nih.gov/>) or European Bioinformatics Institute
(EBI, <https://www.ebi.ac.uk/>). If such a repository is not available,
data can be deposited in unstructured but publicly available
repositories (e.g., Figshare <https://figshare.com/>, Dryad
<https://datadryad.org/>, Zenodo <https://zenodo.org/>, or Open
Science Framework <https://osf.io/>). There are also search
platforms to identify relevant repositories with rigorous standards,
e.g., FairSharing (<https://fairsharing.org/>) and re3data
(<https://www.re3data.org/>).

## Examples

> 'Data Availability: All data are available from Figshare at
<http://dx.doi.org/10.6084/m9.figshare.1288935>'
[@pbio.3000411.ref252].

> 'A fundamental goal in generating this dataset is to facilitate access
to spiny mouse transcript sequence information for external
collaborators and researchers. The sequence reads and metadata are
available from the NCBI (PRJNA342864) and assembled transcriptomes
(Trinity_v2.3.2 and tr2aacds_v2) are available from the Zenodo
repository (<https://doi.org/10.5281/zenodo.808870>), however accessing
and utilizing this data can be challenging for researchers lacking
bioinformatics expertise. To address this problem we are hosting a
SequenceServer... BLAST-search website
(<http://spinymouse.erc.monash.edu/sequenceserver/>). This resource
provides a user-friendly interface to access sequence information from
the tr2aacds_v2 assembly (to explore annotated protein-coding
transcripts) and/or the Trinity_v2.3.2 assembly (to explore non-coding
transcripts)' [@pbio.3000411.ref253].
